createdisk.sh

```diff
@@ -57,7 +57,16 @@ function create_qemu_image {
 
     # Make sure you have enough space in /tmp to run VIRT_SPARSIFY
```

Comment:
Does this comment need to change, since `TMPDIR=$(pwd)/$destDir` is used now?

Reply:
Ah right. This used to be "TMPDIR must point at a directory with as much free space as the size of the image we want to sparsify" before commit f0768092d